test/mocks/BinaryModule/BinaryModule.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <AssemblyName>BinaryModule</AssemblyName>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="PowerShellStandard.Library" Version="5.1.0-preview-06">
```

Comment:
I know it's just a sample but should it be [5.1.1](https://www.nuget.org/packages/PowerShellStandard.Library/5.1.1)?

Reply:
It literally uses what `dotnet new psmodule` template does, so that template should be updated if that's the intent :)

Reply:
What even is that module?! Yes it should probably be updated.

Reply:
I see what happened. `master` has been update to use the 7.x preview but that hasn't been released, so the last released version of `psmodule` template is out-of-date: https://github.com/PowerShell/PowerShellStandard/blob/master/src/dotnetTemplate/Microsoft.PowerShell.Standard.Module.Template/Microsoft.PowerShell.Standard.Module.Template/Microsoft.PowerShell.Standard.Module.Template.csproj

Reply:
I'll update it for now assuming a PR will get that re-released.